x-pack/plugins/observability/public/embeddable/slo/overview/slo_embeddable.tsx

```diff
@@ -50,10 +50,6 @@ export class SLOEmbeddable extends AbstractEmbeddable<SloEmbeddableInput, Embedd
     this.updateInput({ title });
   }
 
-  public reportsEmbeddableLoad() {
```

Comment:
@shahzad31 Why was this removed? We had a bug in the Overview embeddable regarding screenshot reporting https://github.com/elastic/kibana/pull/169929. I had a look at other plugins and here's how [change point chart embeddable](https://github.com/elastic/kibana/blob/main/x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart.tsx#L90) does it.

Reply:
I confirm that this [change](https://github.com/elastic/kibana/commit/a8647151cb69bbd22f5f1100fe0cfc30599a24f8) broke our embeddables which ended up being in an endless loading state.

<img width="1513" alt="Screenshot 2023-11-30 at 12 01 07" src="https://github.com/elastic/kibana/assets/2852703/999ac942-f7cc-45ee-80e8-96088abed4df">


For now I will not add `reportsEmbeddableLoad` back, until I investigate it further. I was not using it anyway. Here's the [way](https://github.com/elastic/kibana/pull/169929/files) I had solved the screenshot reporting issue.